inc/class-wpseo-replace-vars.php

```diff
@@ -1146,15 +1146,85 @@ private function get_replacement_variables() {
 	 *
 	 * @param string $replacement_variable The replacement variable.
 	 *
-	 * @return bool True when the replacement variable contains a
+	 * @return bool True when the replacement variable is not prefixed.
 	 */
 	private function is_not_prefixed( $replacement_variable ) {
-		$prefixes = array( 'cf_', 'ct_' );
-		$prefix   = substr( $replacement_variable, 0, 3 );
+		$prefixes = array( 'cf_', 'ct_', 'pt_' );
+		$prefix   = $this->get_prefix( $replacement_variable );
 
 		return ! in_array( $prefix, $prefixes );
 	}
 
+	/**
+	 * Strip the prefix from a replacement variable name.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The replacement variable name without the prefix.
+	 */
+	private function strip_prefix( $replacement_variable ) {
+		return substr( $replacement_variable, 3 );
+	}
+
+	/**
+	 * Gets the prefix from a replacement variable name.
+	 *
+	 * @param string $replacement_variable The replacement variable.
+	 *
+	 * @return string The prefix of the replacement variable.
+	 */
+	private function get_prefix( $replacement_variable ) {
+		return substr( $replacement_variable, 0, 3 );
+	}
+
+	/**
+	 * Strips 'desc_' if present, and appends '(description)' at the end.
```

Comment:
I am missing the `(` and `)` around description

Reply:
Not addressed ^